gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/hive/HiveDataset.java

```diff
@@ -94,6 +96,7 @@
 
   // Will not be serialized/de-serialized
   protected transient final Properties properties;
+  public Properties getProperties() { return properties; }
```

Comment:
Please move the method into the section below (methods should stay with method. Also you can use lombok annotation for boilerplate code. Check the "@Getter" annotation in the codebase for example)